examples/example_server.py

```diff
@@ -59,7 +59,7 @@ def on_peer_connected(self, peer):
         def on_peer_disconnected(self, peer):
             self.logger.info('Peer disconnected: {}'.format(peer))
 
-        def on_midi_commands(self, peer, command_list):
+        def on_midi_commands(self, peer, command_list, journal):
```

Comment:
How about we update the signature to return `midi_packet` here instead of just `journal`? Possibly there are other/future uses for it.

Since this is an interface change, can you also be sure to:

* Update the generic `Handler` class in `server.py` (line ~21) to match the new signature
* Update the example in `README.md` to match the new signature
* Add an entry to `CHANGELOG.md` under `Current version`, something like: _Breaking change: added `midi_packet` to `on_midi_commands()` callback_
```